$OpenBSD: patch-scapy_tools_generate_ethertypes_py,v 1.1 2020/08/21 18:38:12 bluhm Exp $

In Python 3 br is a synonym of rb.  Python 2 does not know br.

Index: scapy/tools/generate_ethertypes.py
--- scapy/tools/generate_ethertypes.py.orig
+++ scapy/tools/generate_ethertypes.py
@@ -20,7 +20,7 @@ URL = "https://raw.githubusercontent.com/openbsd/src/m
 with urllib.request.urlopen(URL) as stream:
     DATA = stream.read()
 
-reg = rb".*ETHERTYPE_([^\s]+)\s.0x([0-9A-Fa-f]+).*\/\*(.*)\*\/"
+reg = br".*ETHERTYPE_([^\s]+)\s.0x([0-9A-Fa-f]+).*\/\*(.*)\*\/"
 COMPILED = b"""#
 # Ethernet frame types
 #       This file describes some of the various Ethernet
